Sunrise/ViewModels/CartViewModel.swift

```diff
@@ -119,6 +119,17 @@ class CartViewModel: BaseViewModel {
         return cart.value?.lineItems?[indexPath.row].totalPrice?.description ?? "N/A"
     }
 
+    func productDetailsViewModelForLineItemAtIndexPath(indexPath: NSIndexPath) -> ProductViewModel {
+        var product = ProductProjection()
+        if let lineItem = cart.value?.lineItems?[indexPath.row] {
+            product.id = lineItem.productId
+            // When navigating to PDP from LineItem, we only show one variant
+            product.masterVariant = lineItem.variant
+            product.name = lineItem.name
```

Comment:
In case we only show the variant which has been added to cart:
- As you said, one less API call.
- For now, we can use this for simply increasing the quantity.
- IMO, If as I user, I purchased something in one size, it's probably unlikely I'm going to need it in another.

In case we show the full product:
- The good thing is, that it's a full PDP.
- Users in this case can add any other variant to their cart.

I'm going to proceed with this change, and at the same time we should probably add the quantity control for a line item soon ☺️